pkg/k8sclient/client.go

```diff
@@ -124,3 +128,30 @@ func outOfClusterConfig() (*rest.Config, error) {
 	config, err := clientcmd.BuildConfigFromFlags("", kubeconfig)
 	return config, err
 }
+
+// stopCacheReset - Stops the rest mapper cache from reseting.
+func stopCacheReset() {
```

Comment:
Do we ever need to call `stopCacheReset()`? I don't see it being used anywhere.
Also it's not something exposed to the user in the handler since it's not expected that an operator would ever want to turn this off and on again as part of normal operations.

So I would suggest to remove `stopCacheReset()` for now until we have a use case for it.
With that we can also get rid of `quit` and `started` and simplify the cache reset to:
```Go
func init() {
...
  go periodicResetRestMapper(1 * time.Minute)
}

func periodicResetRestMapper(duration time.Duration) {
  ticker := time.NewTicker(duration)
  for {
    select {
    case <-ticker.C:
      restMapper.Reset()
    }
  }
}
```

Ideally we'd pass a context to the go routine but we can't do that while all of this is being done in the init() function. That will have to wait until we refactor all of the global state into objects.